dts/arm/gigadevice/gd32e50x/gd32e50x.dtsi

```diff
@@ -411,6 +411,7 @@
 				     <15 0>, <16 0>, <17 0>;
 			clocks = <&cctl GD32_CLOCK_DMA0>;
 			dma-channels = <7>;
+			mem2mem;
```

Comment:
nit: use vendor prefix for this prop `gd,mem2mem` (we likely need to fix a few others already in-tree)

Reply:
Thank you for pointing out.
I Fixed it.